frontend/src/libs/components/prompt-generation/libs/enums/prompt-category.enum.ts

```diff
@@ -0,0 +1,9 @@
+const PromptCategory = {
```

Comment:
Check `PromptBaseResponseDto` - it's fields is the actual values you're using here - and we may need to restrict any non-synchronized changes - so it would be better to move this enum to shared folder and use it for `PromptBaseResponseDto` fields definition